### DISCUSSION TITLE
CUSA19143 - Disciples Liberation 1.04

### DISCUSSION THREAD

**Discussion by SrPhreaK** [GoldHEN/GoldHEN_Cheat_Repository#36] [Cheat requests]
here is the suggestion of cheats for this game.
anything I'm already grateful.
thanks for the support already given to other games.

last month I also left here the suggestion for
CUSA13760 - Conception Plus Maidens of The Twelve Stars
I hope you pay attention.
and thank you .

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
